js/src/views/Signer/containers/RequestsPage/requestsPage.js

```diff
@@ -85,7 +88,9 @@ class RequestsPage extends Component {
       >
         <TxList
           address=''
+          blockNumber={ blockNumber }
           hashes={ localHashes }
+          newError={ newError }
```

Comment:
Isn't there an agreement that fired events should be always prefixed with `on*`? Imho thinking in terms of `props` and `events` helps to understand components architecture better (without imperative: "`createNewErrorNow` stuff)